rfcs/105-trim-service-pair-prefix.md

```diff
@@ -5,6 +5,9 @@
 
 # GSP-105: Trim Service Pair Prefix
 
+- Updated By:
```

Comment:
Ping @xxchan to confirm the right place for `updated by` & `updates`.

Reply:
LGTM